cli/config_file.rs

```diff
@@ -31,6 +31,14 @@ pub struct EmitConfigOptions {
   pub jsx_fragment_factory: String,
 }
 
+/// There are certain compiler options that can impact what modules are part of
+/// a module graph, which need to be deserialized into a structure for analysis.
+#[derive(Debug, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct CompilerOptions {
+  pub types: Option<Vec<String>>,
+}
```

Comment:
I'm a bit confused now - I thought we already parsed the value of `compilerOptions` into a concrete structure. Isn't that the case?

Reply:
For `Deno.emit()` no, we pass a `Value` from the op when we merge it into the config.